### PR TITLE
Fixed lmo_pairwise when the 0 vertex is chosen by the Away step

### LINKED ISSUE #75: pairwise_lmo outputs 0 update_direction
This breaks the 'DR' step scheme, which is proportional to`1/norm(update_direction)`.
It happens even when we are not at the optimum.

In particular, where does [this](https://github.com/openopt/copt/blob/96d3c21bbd8f38b7a09fe21b8689b4d381ba1c7a/copt/constraint.py#L60) part of the code come from?

### PR DESCRIPTION
**Pull request by GeoffNN**
An attempt to fix #75

### CODE REVIEW

File: copt/constraint.py [hunk 1]
```diff
@@ -47,23 +47,20 @@ def lmo_pairwise(self, u, x):
         )
 
         u_active = -u * np.sign(x)
-        ma_u_active = ma.array(u_active, mask=(u_active == 0))
+        ma_u_active = ma.array(u_active, mask=(x== 0) + (u_active==0))
```
Comment (fabianp) [inline copt/constraint.py:50]: I'm not entirely convinced by this. What if the domain does not include zero? I think the issue has more to do with how to include x0 as a vertex so we start with atom in the support than with zero

**Comment by fabianp** [comment]
Could you add some tests for this so we don't break it again? thanks!

**Comment by GeoffNN** [comment]
Not entirely convinced myself -- still working on it.

This method is for the L1 ball, so it always includes the 0 point. The 0 "vertex", is because you can always write the iterate as a convex combo of 0 and the +/- e_i vectors.

**Comment by fabianp** [comment]
I see, I missed that part. I'm fine with this solution as long as it has some tests then :-)

**Comment by GeoffNN** [comment]
The previous commit doesn't work as such: the iterate can go out of the constraint set.

**Pull request closed without merging**